docs/markdown/Helm/helm-deployments.md

```diff
@@ -290,7 +290,7 @@ helm_deployment(
 )
 ```
 
-In the previous example we define a `experimental_shell_command` target that will invoke the `vals eval` command (`vals` needs to be installed in the local machine) as part of the Helm post-rendering machinery, which will result on the `ref+vault` reference being replaced by the actual value stored in Vault at the given path.
+In the previous example we define a `shell_command` target that will invoke the `vals eval` command (`vals` needs to be installed in the local machine) as part of the Helm post-rendering machinery, which will result on the `ref+vault` reference being replaced by the actual value stored in Vault at the given path.
```

Comment:
Both old and new versions of these docs seem to be ... not quite right:
```suggestion
In the previous example we define a `experimental_run_shell_command` target that will invoke the `vals eval` command (`vals` needs to be installed in the local machine) as part of the Helm post-rendering machinery, which will result on the `ref+vault` reference being replaced by the actual value stored in Vault at the given path.
```

Reply:
Grumble, clicked _merge_ before spotting this. Will follow up shortly.